pygments/lexers/templates.py

```diff
@@ -2264,3 +2266,18 @@ class Angular2HtmlLexer(DelegatingLexer):
 
     def __init__(self, **options):
         super().__init__(HtmlLexer, Angular2Lexer, **options)
+
+
+class SqlJinjaLexer(DelegatingLexer):
+    """
+    Templated SQL lexer.
+
+    .. versionadded:: 2.13
+    """
+
+    name = 'SQL+Jinja'
+    aliases = ['sql+jinja']
+    filenames = ['*.sql']
```

Comment:
This is problematic because the plain `SqlLexer` already declares `*.sql`. If you want this, you need to add an `analyse_text()` method and check the one defined by `SqlLexer` so that they will distinguish.

Also, the other Jinja2 lexers have `*.xxx.j2` and `*.xxx.jinja2` filenames declared. I'd add these.

Reply:
Hi @Jean-Abou-Samra! Thanks for the review and suggestion. I've implemented `analyse_text` and added some unit tests too.